community_images/metrics-server/ironbank/k8s_coverage.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# Getting Runtime Parameters
+
+JSON_PARAMS="$1"
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json Params for k8s coverage = $JSON"
+
+NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
+RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
+POD_NAME=$(kubectl get pod -n "${NAMESPACE}" | grep "${RELEASE_NAME}-[a-z0-9-]*" --color=auto -o)
```

Comment:
Can you please verify if `--color=auto` is required?
If not, can you please remove it?

Reply:
removed` --color=auto`